test/lit.cfg

```diff
@@ -657,6 +657,56 @@ elif run_os == 'linux-gnu':
         "ld -L%s" %
         (os.path.join(test_resource_dir, config.target_sdk_name)))
 
+elif run_os == 'linux-gnueabihf':
```

Comment:
What is the difference between this block and the previous block?  Can it be factored with 'if' statements instead of code duplication?